client/state/site-settings/exporter/reducers.js

```diff
@@ -33,19 +33,31 @@ export function selectedPostType( state = null, action ) {
 	return state;
 }
 
-export function exportingState( state = States.READY, action ) {
-	switch ( action.type ) {
-		case REQUEST_START_EXPORT:
-			return States.STARTING;
-		case REPLY_START_EXPORT:
-			return States.EXPORTING;
-		case FAIL_EXPORT:
-		case COMPLETE_EXPORT:
-			return States.READY;
+/**
+ * Tracks the state of the exporter for each site ID
+ * @param  {Object} state  The current state
+ * @param  {Object} action Action object
+ * @return {Object}        Updated state
+ */
+export function exportingState( state = {}, { type, siteId } ) {
+	switch ( type ) {
+		case EXPORT_START_REQUEST:
+			return Object.assign( {}, state, {
+				[ siteId ]: States.STARTING
+			} );
+		case EXPORT_STARTED:
+			return Object.assign( {}, state, {
+				[ siteId ]: States.EXPORTING
+			} );
+		case EXPORT_COMPLETE:
+		case EXPORT_FAILURE:
+			return Object.assign( {}, state, {
+				[ siteId ]: States.READY
+			} );
 		case SERIALIZE:
 			return state;
 		case DESERIALIZE:
-			return States.READY;
+			return null;
```

Comment:
It also, looks like we're persisting state in selectedPostType above. If you don't want to persist that value, return null for both `SERIALIZE` and `DESERIALIZE` since that's the default initial value.